storage/client_range_test.go

```diff
@@ -150,3 +155,156 @@ func TestRejectFutureCommand(t *testing.T) {
 		t.Errorf("expected 15, got %v", v)
 	}
 }
+
+// Test a case where a put operation of an older timestamp comes after
+// a put operation of a newer timestamp.
```

Comment:
Thanks for writing this test! I'd appreciate it if you can reword this comment to disambiguate between the reader and the writer. You use txn a lot in this comment. Perhaps replacing it with Reader and Writer might help clarify what this test is testing. Thanks as always